txn_writer.go

```diff
@@ -152,7 +152,7 @@ func (sw safeWriter) writeAllSafe(forceVendor bool) error {
 
 	if writeM {
 		if _, err := os.Stat(mpath); err == nil {
-			// move out the old one
```

Comment:
I think you are talking about side notes such as

```
c.cancel(true, DeadlineExceeded) // deadline has already passed
```

Comments on their own line are proper sentences.

```
       if cur, ok := parent.Deadline(); ok && cur.Before(deadline) {
		// The current deadline is already sooner than the new one.
		return WithCancel(parent)
	}
```